### PR TITLE
Update to hsqldb 2.5.1

### PR DESCRIPTION
**Pull request by amarkevich**

### CODE REVIEW

File: rt/rs/security/sso/oidc/src/main/java/org/apache/cxf/rs/security/oidc/idp/OidcUserSubject.java [hunk 1]
```diff
@@ -54,6 +55,7 @@ public OidcUserSubject(UserSubject sub) {
 
     @Lob
     @Basic(fetch = FetchType.EAGER)
+    @Column(length = 1024)
```
Comment (coheigea) [inline rt/rs/security/sso/oidc/src/main/java/org/apache/cxf/rs/security/oidc/idp/OidcUserSubject.java:58]: Where does 1024 come from?

Comment (amarkevich) [reply rt/rs/security/sso/oidc/src/main/java/org/apache/cxf/rs/security/oidc/idp/OidcUserSubject.java:58]: Looks like recent Hibernate cause [1]: 

> creates column as CLOB(255) 

So I set some higher value.

1. https://stackoverflow.com/a/26805926/3476615

Comment (coheigea) [reply rt/rs/security/sso/oidc/src/main/java/org/apache/cxf/rs/security/oidc/idp/OidcUserSubject.java:58]: I'm wondering if a JWT token can be longer than 1024 characters....say if it has a lot of claims?

Comment (amarkevich) [reply rt/rs/security/sso/oidc/src/main/java/org/apache/cxf/rs/security/oidc/idp/OidcUserSubject.java]: updated @Column annotation: explicit 'blob' column type without length

**Pull request merged**